packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -100,3 +126,76 @@ export const instructionToCommand = <Input, ContractInput>(instruction: Abstract
     }
   }
 }
+
+export const instructionToBatchCommand = <Input, ContractInput>(
```

Comment:
This will require having an explicit instruction for every command again to make it a batch Command. Why don't we just accept the command as input and generate the batch command with it, similar to the Multisig.